villainary-react/app/interfaces/api-url.ts

```diff
@@ -0,0 +1 @@
+export type ApiUrl = 'VillainaryProducts'
```

Comment:
is a type best here?  As i understand it, when we add more endpoints, we will need to use a union type to do so.  I think that could get quite messy with multiple endpoints listed such as

 `export type ApiUrl = 'VillainaryProducts' | 'VillainaryHenchmen' | 'VillainaryLairs; `.  
 
 I think I'd prefer the enum approach:

  `export enum ApiUrl {
    VillainaryProducts = 'VillainaryProducts',
    VillainaryHenchmen = 'VillainaryHenchmen',
}`


What do you think?

Reply:
I personally don't see the point in making the enum. The union provides just as good intellisense, and is shorter.